packages/core/src/components/tabs/tabs.tsx

```diff
@@ -98,13 +109,19 @@ export class Tabs extends AbstractPureComponent<ITabsProps, ITabsState> {
     public static displayName = `${DISPLAYNAME_PREFIX}.Tabs`;
 
     private tablistElement: HTMLDivElement;
+    private overflowElement: React.Component<ITabsProps, IOverflowListState<any>>;
     private refHandlers = {
+        overflowList: (tabElement: React.Component<ITabsProps, IOverflowListState<any>>) =>
+            (this.overflowElement = tabElement),
         tablist: (tabElement: HTMLDivElement) => (this.tablistElement = tabElement),
     };
 
     constructor(props?: ITabsProps) {
         super(props);
         const selectedTabId = this.getInitialSelectedTabId();
+        this.overflowElement = null;
+
+        this.overflowRenderer = this.overflowRenderer.bind(this);
```

Comment:
no! use fat arrows in the definition.

`private overflowRenderer = (args) => // bound`